receiver/huaweicloudcesreceiver/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+type: huaweicloudcesreceiver
+
+status:
+  class: receiver
+  stability:
+    development: [metrics]
+  distributions: []
+  codeowners:
+    active: [heitorganzeli, narcis96]
```

Comment:
Should this list include the sponsor as well?

Reply:
done.